src/displayapp/DisplayApp.cpp

```diff
@@ -208,7 +208,9 @@ void DisplayApp::RunningState() {
       case Apps::SysInfo: currentScreen.reset(new Screens::SystemInfo(this, dateTimeController, batteryController, brightnessController, bleController, watchdog)); break;
       case Apps::Meter: currentScreen.reset(new Screens::Meter(this)); break;
       case Apps::Twos: currentScreen.reset(new Screens::Twos(this)); break;
-      case Apps::Gauge: currentScreen.reset(new Screens::Gauge(this)); break;
+
+      //case Apps::Gauge: currentScreen.reset(new Screens::Gauge(this)); break;
```

Comment:
Why is Gauge not working properly?

Reply:
The Gauge is not used, I did not make any corrections to the new version.

Reply:
Okok just commenting so it will be removed before this get merged. Same with the places you have added "joaquimorg to FIX". Shouldn't Gauge and Modal get removed completely?